diagnostic.md

```diff
@@ -32,22 +35,26 @@ Give two examples of jQuery getter methods and two examples of jQuery setter
 methods.
 
 <!-- your answer starts here -->
-
+$("#books").html("Hello word") setter
+$("#books").text("lord of the rings") stter
+$("#books").attr()getter
+$("#books").val() getter
 <!-- your answer ends here -->
 
 ## Question 5
 
 In your own words, please answer the following question: WHAT IS THE DOM?
 
 <!-- your answer starts here -->
-
+DOM is the representation of the html on browser.
```

Comment:
The DOM is a language agnostic API for representing HTML documents as a tree structure.  A web browser implements this interface and renders it for you to view and interact with.